src/js/constraint.js

```diff
@@ -8,11 +8,16 @@ const BOUNDS_FORMAT = ['left', 'top', 'right', 'bottom'];
 
 /**
  * Returns an array of bounds of the format [left, top, right, bottom]
+ * @param body
  * @param tether
  * @param to
  * @return {*[]|HTMLElement|ActiveX.IXMLDOMElement}
  */
 function getBoundingRect(body, tether, to) {
+  // arg to is required
+  if (!to) {
```

Comment:
I could be wrong, but if `to` is undefined or null in the current code, I believe this should just return those values. Do we need this check?

Reply:
my thought was return early if not defined, just a little clearer. Can remove if you're opposed.

Reply:
Since I think it works without this change, I would advocate for removing. Less code is best.

Reply:
If `to` is undefined, it's fails on this check, `if (!isUndefined(to.nodeType))`. So, it doesn't work without this and we don't enforce it.